packages/@expo/cli/src/export/__tests__/resolveOptions-test.ts

```diff
@@ -20,12 +20,6 @@ describe(resolveOptionsAsync, () => {
     );
   });
 
-  it(`asserts not-configured platform`, async () => {
-    await expect(resolveOptionsAsync('/', { '--platform': ['web'] })).rejects.toThrow(
-      /^Platform "web" is not configured to use the Metro bundler in the project Expo config\./
-    );
-  });
-
```

Comment:
I removed this test as I'm unsure of its functionality. `web` is not configured to use Metro, so this is no longer expected to fail?